zds/settings.py

```diff
@@ -412,6 +410,14 @@
             'email_ca': u"ca-zeste-de-savoir@googlegroups.com",
             'forum_ca_pk': 25
         },
+        'repository': {
+            'url': u'https://github.com/zestedesavoir/zds-site',
+            'bugtracker': u'https://github.com/zestedesavoir/zds-site/issues',
+            'api': u'https://api.github.com/repos/GerardPaligot/zds-site',
```

Comment:
Fail de ma part. [C'était pour tester](https://github.com/GerardPaligot/zds-site/issues?q=is%3Aissue+is%3Aclosed).